hooks/post/11-create_sslconf.rb

```diff
@@ -0,0 +1,7 @@
+# post hook to create an ssl.conf file to work-around this issue http://projects.theforeman.org/issues/16972 until mod_ssl packging is fixed.
+
+if File.file?('/etc/httpd/conf.d/ssl.conf') == false
+  File.open('/etc/httpd/conf.d/ssl.conf', 'w') { |file| file.write("# This file is managed by the foreman-installer, do not alter.") }
+else
+  logger.info 'ssl.conf is already present, skipping'
+end
```

Comment:
Looks fine, a few ruby style things though.

Generally I think "if false + else" isn't a good code practice - harder to read. It's better to swap the conditions (although this is really a matter of opinion):

```ruby
if File.file?('/etc/httpd/conf.d/ssl.conf')
  logger.info 'ssl.conf is already present, skipping'
else
  File.open('/etc/httpd/conf.d/ssl.conf', 'w') { |file| file.write("# This file is managed by the foreman-installer, do not alter.") }
end
```
But at the very least, don't compare with false. Some methods won't return an actual false object but rather something else that is falsy like "nil".  It's very common in Rails itself that some things return nil instead of false.

An example:

```ruby
a = nil

if !a
  puts "this will print"
end

if a == false
  puts "this won't"
end
```

File.file? does return false, but it's a bad habit.  IMHO prefer "!File.file('foo')" instead, or avoid the negation altogether by swapping the conditions like the first suggestion